package.json

```diff
@@ -22,19 +22,38 @@
     "posttest": "yarn lint",
     "lint": "yarn cmsds lint ./ --ignorePatterns '**/node_modules/**' '**/dist/**' '**/helpers/**' '**/__tests__/**' 'tmp/**' '**/types/**' 'examples/create-react-app-typescript/**' 'examples/create-react-app/**'",
     "prepare": "husky install",
-    "gh-pages": "yarn build:healthcare && gh-pages -d './packages/ds-healthcare-gov/docs/dist' --dest 'healthcare'"
+    "gh-pages": "yarn build:healthcare && gh-pages -d './packages/ds-healthcare-gov/docs/dist' --dest 'healthcare'",
+    "storybook": "start-storybook -p 6006 -s ./packages/design-system/src",
```

Comment:
`-s ./packages/design-system/src` serves the files at that location as static assets for storybook to reference.
I did this so we could load our fonts properly, but I don't feel great about serving the whole src directory as static assets.

I'm not sure if this is something I should worry about (like for performance), so open to thoughts or ideas